### ISSUE TITLE
session_set_save_handler error in php7

### ISSUE THREAD

**Issue by lujunjie521** [phalcon/cphalcon#11733]
when use memcache,redis to save session, but `session_set_save_handler->write` has a bug,
`function write($id, $data)` need bool return, but in the source miss the return :+1: 

/phalcon/session/adapter/redis.zep line:125
```zephir
public function write(string sessionId, string data)
{
    // this coding don't have the value return
    this->_redis->save(sessionId, data, this->_lifetime); 
}
```
so ,i think the follow coding maybe right:
```zephir
public function write(string sessionId, string data)
{
    this->_redis->save(sessionId, data, this->_lifetime);
    // just return true cause of the redis cache save
    // method don't have the status of save return
    return true;
}
```

**Comment by KennethVeipert** [comment]
I can reproduce this bug, if i create a class that extends \Phalcon\Session\Adapter\Redis and overwrite method `write`:
```php
	/**
	 * @param string $sessionId
	 * @param string $data
	 * @return bool
	 */
	public function write($sessionId, $data)
	{
		parent::write($sessionId, $data);

		return true;
	}
```
everything works fine.

To reproduce:
I use this to initialize the session with redis-adapter:
```php
$session = function () use ($configuration): \Phalcon\Session\Adapter {
	$session = new \Phalcon\Session\Adapter\Redis([
		'uniqueId'   => 'some-uniqueID',
		'lifetime'   => 2592000,
		'prefix'     => 'my-prefix-',
		'host'       => $host,
		'port'       => $port,
		'auth'       => $password,
		'persistent' => true,
	]);

	$session->start();

	return $session;
};

// Do some other, irrelevant stuff

$this->di->set('session', $session);
```
Then you can see at the bottom of the page:
```html
<b>Warning</b>:  session_write_close(): Session callback expects true/false return value in <b>Unknown</b> on line <b>0</b><br />
<br />
<b>Warning</b>:  session_write_close(): Failed to write session data (user). Please verify that the current setting of session.save_path is correct (/var/lib/php/sessions) in <b>Unknown</b> on line <b>0</b><br />
```

Hope this helps.

Phalcon-Version: 2.0.11
PHP: 7.0.4-7ubuntu2.1

**Comment by sergeyklay** [comment]
Fixed in the `2.1.x` branch

**Comment by cq-z** [comment]
PHP Warning:  session_destroy(): Session callback expects true/false return value
redis->delete($id) renturn not bool

Documentation ：
```
Return value

Long Number of keys deleted.
```

**Comment by sergeyklay** [comment]
I'll check

**Comment by sergeyklay** [comment]
Fixed in the `2.1.x` branch
